README.md

```diff
@@ -132,7 +132,8 @@ To set options and keep lint-staged extensible, advanced format can be used. Thi
 
 ## Filtering files
 
-It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns. The paths used for filtering via minimatch are relative to the directory that contains the `.git` directory. The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory, as would be the case when using the `gitDir` option.
```

Comment:
I think

> The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory

is important part here in order to reduce questions like #369

Reply:
My bad. I removed the line because it was mentioning `gitDir`.